src/NuGet.Core/NuGet.CommandLine.XPlat/ReportRenderers/Models/ListPackageOutputContentV1.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+
+namespace NuGet.CommandLine.XPlat
+{
+    /// <summary>
+    /// List package json output version 1
+    /// </summary>
+    internal class ListPackageOutputContentV1
+    {
+        internal int Version { get; set; } = (int)ReportOutputVersion.V1;
+        internal ListPackageArgs ListPackageArgs { get; set; }
+        internal List<ReportProblem> Problems { get; set; }
+        internal List<ListPackageProjectModel> Projects { get; set; }
+        internal bool AutoReferenceFound { get; set; }
```

Comment:
nit: Isn't this contained *within* a project report, so it should probably be a part of that?

Reply:
You are mixing the view and the model. 

The model is that a project has an autoreferenced package. 
The view formats the columns different based on that.

Reply:
Ok. Moved to project model.